packages/trezor-user-env-link/e2e/api.test.ts

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-console */
+
+import { typedObjectKeys } from '@trezor/utils';
+
+import { TrezorUserEnvLink } from '../src';
+
+(async () => {
+    await TrezorUserEnvLink.connect();
+
+    if (!TrezorUserEnvLink.firmwares) {
+        throw new Error('firmwares not loaded');
+    }
+    for (const model of typedObjectKeys(TrezorUserEnvLink.firmwares)) {
+        console.log('testing firmwares for model', model);
+
+        for (const version of TrezorUserEnvLink.firmwares[model]) {
+            console.log('starting emulator', model, version);
+            await TrezorUserEnvLink.startEmu({
+                model,
+                version,
+                wipe: true,
+            });
+
+            console.log('setting up emulator');
+            await TrezorUserEnvLink.setupEmu();
+
+            await TrezorUserEnvLink.allowUnsafePaths();
+            // todo: run more complex commands, typically those that require interaction with multiple screens (dry run..)
```

Comment:
I need to get device somehow into recovery/backup state. Can we do it with current tenv API? I can't see anything. If no, could you please either expose some trezorlib method which allows this, or maybe if no such method exists I would be ok with passing raw bytes. cc @grdddj (no hurry here)